### DISCUSSION TITLE
Pixeval 路线图

### DISCUSSION THREAD

**Discussion by Poker-sang** [Pixeval/Pixeval#345] [Announcements]
- [x] 将用户页面单独弹出窗口，并支持任何地方点击头像都能弹出窗口
- [x] 优化标题栏样式，等待[TitleBar](https://github.com/CommunityToolkit/Labs-Windows/discussions/454)控件发布
- [x] 添加用户简介，等待[MarkdownTextBlock](https://github.com/CommunityToolkit/Labs-Windows/pull/480)控件发布
- [x] 添加小说功能
- [x] 添加特辑（pixivision）api及页面
- [x] 悄悄收藏功能，收藏分类功能
- [ ] 关于页面添加赞助者
- [ ] 等待[NavigationView漏内存问题](https://github.com/microsoft/microsoft-ui-xaml/issues/5978)修复，减少内存使用
- [x] 监视注册表，在更改的代理的时候更新HttpClient，以实现“与系统代理同步”效果
- [x] 更简单的安装方式

**Comment by dylech30th** [comment]
Additional major improvements

- [ ] Enhanced tag/local database system, this is still under prototyping, any suggestions are welcome (Can we make the My Tag page less technical, more nice-looking, and more accessible?)
- [x] User feeds (*Prioritized!*)
- [ ] A download progress notifier in IllustrationViewer
- [ ] Tag suggestions (cf. this on the index page of pixiv.net)
![image](https://github.com/Pixeval/Pixeval/assets/31894334/0f17f538-5841-41c3-883f-0814b1129079)
- [x] Remove one of the suggestion boxes, unified UI experience
- [ ] Index page
- [x] Help message on macros
- [ ] Notify users through toast notification when new feeds are available
- [ ] Can we do better on the History page? That is, each item on the history page enjoys an entry showing the time at which we visited it
- [ ] Can we make the reverse search less technical? That is, user can use it without setting an API key?
- [x] Auto updater! (Don't forget the "metered network" option!)

### Long-term goals
- [x] Overall UX experience improvements: How to make filters and macros less technical, more nice-looking, and more accessible? Are there existing examples?

**Comment by Poker-sang** [comment]
## 更多目标

- [ ] AOT
- [x] 插件系统
- [x] 自动翻译功能
- [x] 接入Crowdin
- [ ] 多用户
- [x] 登陆前修改设置

**Comment by Poker-sang** [comment]
- [x] Markdown/RTF翻译
- [x] Tab关闭卡顿
- [ ] 筛选补全
- [x] danbooru api接入（抽象接口）
- [ ] 下载插件
- [x] 红心动画

**Comment by Poker-sang** [comment]
- [x] 自动生成代理证书
- [ ] 自定义EXIF信息
